EIPS/eip-4804.md

```diff
@@ -0,0 +1,65 @@
+---
+eip: 4804
+title: Exodus
+description: Exodus allows to create blockchain networks with accounts and transactions.
+editors: Antoine Toulmé (@atoulme), Chaals Nevile (@chaals)
+discussions-to: <https://entethalliance.org/contact>, For EEA members only (requires a patent commitment): <https://github.com/entethalliance/testing-wg/issues> 
```

Comment:
This should be a link to a thread on the Ethereum Magicians forum.

Reply:
We don't intend to use the Ethereum Magicians forum for discussion.

Reply:
While I think Sam's question is useful to answer and I'm also curious, all EIP discussions-to links must be to Ethereum Magicians.